resources/views/auth/login.blade.php

```diff
@@ -0,0 +1,60 @@
+@extends('layouts.app')
+
+@section('content')
+    <div class="login-section section pt-95 pt-lg-75 pt-md-65 pt-sm-55 pt-xs-45">
+        <div class="container">
+            <div class="row">
+                <div class="col-md-6 col-12 d-flex">
+                    <div class="gilbard-login">
+                        
+                        <h3>{{ trans('text.login.login_to') }}</h3>
+                        <p>{{ trans('text.shopping') }} <i class="fa fa-gamepad"></i></i></p>
+
+                        <form method="POST" action="{{ route('login') }}">
+                            @csrf
+                            <div class="row">
+                                <div class="col-12 mb-30">
+                                    <input id="email" class="@error ('email') is-invalid @enderror" type="email" name="email" placeholder="{{ trans('text.login.type_your_email_address') }}" value="{{ old('email') }}" required autocomplete="email" autofocus>
+                                    @error('email')
+                                        <span class="invalid-feedback" role="alert">
+                                            <strong>{{ $message }}</strong>
+                                        </span>
+                                    @enderror
+                                </div>
+
+                                <div class="col-12 mb-20">
+                                    <input id="password" type="password" name="password" placeholder="{{ trans('text.login.enter_your_password') }}" class="@error ('password') is-invalid @enderror" name="password" required autocomplete="current-password">
+                                    @error('password')
```

Comment:
```suggestion
                                    @error ('password')
```
sửa hết mấy chỗ tương tự đi nhá